src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1349,13 +1351,22 @@ private function getServiceCall($id, Reference $reference = null)
      */
     private function camelize($id)
```

Comment:
this private method should be renamed though, as it is not about camelizing anymore, but about generating the method name